.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
+.venv
+result.png
```

Comment:
If result.png is not needed in the Docker image for the application to function, it's correctly ignored here.